albumaction/albumact.rb

```diff
@@ -0,0 +1,86 @@
+require 'json'
+require_relative '../music'
+
+class Albumaction
+  def getmusicalbum
+    file_path = './album_data/music.json'
+    if File.exist?(file_path)
+      file_content = File.read(file_path)
+      JSON.parse(file_content)
```

Comment:
[OPTIONAL] These two lines are already in line numbers 13 and 14, so they can be deleted